mycore-base/src/main/java/org/mycore/frontend/servlets/MCRServlet.java

```diff
@@ -289,6 +304,10 @@ private static boolean isSessionBoundToRequest(HttpServletRequest req) {
      */
     private void doGetPost(HttpServletRequest req, HttpServletResponse res) throws ServletException, IOException,
         SAXException, TransformerException {
+        if (disabled) {
+            res.sendError(HttpServletResponse.SC_NOT_IMPLEMENTED, "This servlet is disabled.");
+            return;
+        }
```

Comment:
if init() fails with UnavailableException doGetPost() is never called, so here no check is needed.